includes/wpum-forms/class-wpum-form-registration.php

```diff
@@ -532,7 +532,7 @@ public function done() {
 	 */
 	public function render_registration_form_fields($field, $key){
 		$registered_groups = array_column(wpum_get_registered_field_types(), 'fields');
-		$registered_fields = $registered_groups ? array_merge(...$registered_groups) : [];
+		$registered_fields = $registered_groups ? call_user_func_array('array_merge', $registered_groups) : [];
```

Comment:
array_reduce($registered_groups, 'array_merge', []));

would this not be the < php 5.6 equivalent? https://www.php.net/manual/en/function.array-merge.php#121023

Reply:
> array_reduce($registered_groups, 'array_merge', []));
> 
> would this not be the < php 5.6 equivalent? https://www.php.net/manual/en/function.array-merge.php#121023

Both works similarly. So i used this one. Workflows different for each one. So i dont think it is a big issue. If you still want it to be changed to `array_reduce` i can replace it

Reply:
That's fine